test/e2e/framework/framework.go

```diff
@@ -73,8 +75,9 @@ type Framework struct {
 	// test multiple times in parallel.
 	UniqueName string
 
-	ClusterClients []*kubeclientset.Clientset
-	federator      []federate.Federator
+	ClusterClients    []*kubeclientset.Clientset
+	LighthouseClients []*lighthouseClientset.Clientset
```

Comment:
You're no longer using this so I think we should remove it. We can always add it later if needed (you can stash it locally).

Reply:
Done.